### PR TITLE
[WIP] fix loss oom

### PR DESCRIPTION
**Pull request by lilei199908**
before optimize
image
after optmize
image
grad norm
image

### CODE REVIEW

File: slime/utils/ppo_utils.py [hunk 1]
```diff
@@ -646,32 +647,38 @@ def chunked_gae(
...
-                entropy = torch.cat(entropys, dim=0)
         else:
-            log_prob = compute_log_probs(logits.clone(), tokens, tp_group)
             if with_entropy:
-                entropy = compute_entropy_from_logits(logits.clone(), tp_group)
+                with entropy_ctx:
+                    entropy_input = logits.clone() if need_entropy_grad else logits
+                    entropy = compute_entropy_from_logits(entropy_input, tp_group)
+
+            log_prob = compute_log_probs(logits.clone(), tokens, tp_group)
```
Comment (zhuzilin) [inline slime/utils/ppo_utils.py:681]: I wonder if some of the code movement (moving log_prob down here) is necessary. And also if we don't need to calculate the grad of entropy, do we still need to do `logits.clone()`?
